src/cmds/scripts/pbs_postinstall.in

```diff
@@ -882,28 +913,28 @@ if [ ! -d "$newpbs_exec" ]; then
 	exit 1
 fi
 
-if [ -z "$6" -o "$6" != "sameconf" ]; then
+if [ "$sameconf" != "true" ]; then
```

Comment:
Depending on how pbs_postinstall is called, it looks for "sameconf" as the last parameter. The number of parameters will vary depending on which RPM is being installed. I updated the code to account for this where previously it did not. It was intentional.